microblocks-adapter.js

```diff
@@ -141,85 +141,111 @@ class MicroBlocksAdapter extends Adapter {
-                                } else if (message.indexOf('moz-property-changed ') === 0) {
-                                    var json = JSON.parse(message.split('moz-property-changed ')[1]);
+                                } else if (message.indexOf(
+                                    'moz-property-changed ') === 0) {
+                                    const json = JSON.parse(message.split(
+                                        'moz-property-changed ')[1]);
                                     console.log('property changed: ', json);
-                                    // second parameter asks to not notify this update back to µBlocks
-                                    device.properties.get(json.name).setValue(json.value, true);
-                                } else if (message.indexOf('moz-property ') === 0) {
-                                    var json = JSON.parse(message.split('moz-property ')[1]);
+                                    // second parameter asks to not notify this
+                                    // update back to µBlocks
+                                    device.properties.get(json.name).setValue(
+                                        json.value, true);
+                                } else if (message.indexOf(
+                                    'moz-property ') === 0) {
+                                    const json = JSON.parse(message.split(
+                                        'moz-property ')[1]);
                                     console.log('got property: ', json);
-                                    deviceDescriptor.properties[json.name] = json;
+                                    deviceDescriptor.properties[json.name] =
+                                        json;
                                 } else if (message.indexOf('moz-done') === 0) {
-                                    device = myself.addDevice(serialPort, deviceDescriptor, protocol);
-                                    serialPort.write(protocol.packMessage('broadcast', 0, protocol.packString('moz-paired')));
+                                    device = myself.addDevice(
+                                        serialPort, deviceDescriptor, protocol);
+                                    serialPort.write(protocol.packMessage(
+                                        'broadcast',
+                                        0,
+                                        protocol.packString('moz-paired')));
                                 } else {
-                                    console.log('received unknown message: ' + message);
+                                    console.log('received unknown message:',
+                                                message);
                                 }
                             } catch (err) {
                                 console.log(err);
                             }
-                        }
+                        },
```

Comment:
What's the reason for adding commas after the last element of a dictionary? You only add them sometimes and I can't see the pattern.

Reply:
Trailing commas make updating objects/arrays easier in the future, as you don't have to remember to add a comma if you're adding a new element at the end.